icons/candy-cane.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11.7 5.8c2.3-2.3 6.1-2.3 8.5 0s2.3 6.1 0 8.5c-.8.8-2 .8-2.8 0-.8-.8-.8-2 0-2.8.8-.8.8-2 0-2.8-.8-.8-2-.8-2.8 0l-9.2 9.8c-.8.8-2 .8-2.8 0-.8-.8-.8-2 0-2.8l9.1-9.9z" />
+  <line x1="6" x2="6" y1="17.8" y2="12" />
+  <line x1="10" x2="10" y1="13.6" y2="7.7" />
+  <line x1="14" x2="14" y1="9.3" y2="4.4" />
+  <line x1="18" x2="18" y1="4.5" y2="10" />
+</svg>
```

Comment:
All in all, this looks pretty nice, but I feel like it's a bit too small in comparison to some related icons.
I made this alternate design using 1by2 and 2by1 diagonals, what do you think?

![image](https://user-images.githubusercontent.com/17746067/234037058-7ea307e6-fc9b-44cd-abb1-40848022a00e.png)

Reply:
Second one looks good… Keeping the `path` to yourself? :)

Reply:
Still had to do some optimizations on it first. :)